MarvelApiClient/src/test/java/com/karumi/marvelapiclient/CharacterApiClientTest.java

```diff
@@ -86,7 +87,7 @@
         .withLimit(ANY_LIMIT)
         .withOffset(ANY_OFFSET)
         .withModifiedSince(getAnyDate())
-        .withOrderBy(CharactersQuery.OrderBy.NAME, false)
+        .withOrderBy(OrderBy.NAME, false)
```

Comment:
You have a non used method in ``CharactersQuery`` you should connect with the new one. This is the method:

````
 public Builder withOrderBy(OrderBy orderBy) {
      this.orderBy = orderBy;
      this.orderByAscendant = true;
      return this;
    }
```